Aspire Budgeting/Views/AspireMasterView.swift

```diff
@@ -10,18 +10,35 @@ struct AspireMasterView: View {
   @EnvironmentObject var sheetsManager: GoogleSheetsManager
   @EnvironmentObject var appCoordinator: AppCoordinator
 
-  @State private var selectedSegment = 0
+  let tabBarItems = [TabBarItem(imageName: "rectangle.grid.1x2", title: "Dashboard"),
+                     TabBarItem(imageName: "creditcard", title: "Accounts"),
+                     TabBarItem(imageName: "arrow.up.arrow.down", title: "Transactions"),
+                     TabBarItem(imageName: "gear", title: "Settings")]
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/Views/AspireMasterView.swift#L16"></span>Multi-line collection literals should have trailing commas.<br />
<code>trailing_comma</code> <code>AspireMasterView.swift:16</code></td>
    </tr>
  </tbody>
</table>